src/domains/transaction/components/MultiSignatureDetail/SentStep.tsx

```diff
@@ -9,16 +9,21 @@ import { Signatures } from "./Signatures";
 export const SentStep = ({
 	transaction,
 	wallet,
+	isBroadcast,
 }: {
 	transaction: DTO.ExtendedSignedTransactionData;
 	wallet: Contracts.IReadWriteWallet;
+	isBroadcast?: boolean;
```

Comment:
Maybe we could make this required as SentStep has only one usage where `isBroadcast` gets passed
```suggestion
	isBroadcast: boolean;
```